test/swoosh/adapters/amazonses_test.exs

```diff
@@ -0,0 +1,127 @@
+defmodule Swoosh.Adapters.AmazonSesTest do
+  use AdapterCase, async: true
+
+  import Swoosh.Email
+  alias Swoosh.Adapters.AmazonSes
+
+  @success_response """
+  <SendEmailResponse>
+    <SendEmailResult>
+      <MessageId>messageId</MessageId>
+    </SendEmailResult>
+    <ResponseMetadata>
+      <RequestId>requestId</RequestId>
+    </ResponseMetadata>
+  </SendEmailResponse>
+  """
+
+  @error_response """
+  <ErrorResponse>
+    <Error>
+      <Type>ErrorType</Type>
+      <Code>ErrorCode</Code>
+      <Message>Error Message</Message>
+    </Error>
+    <RequestId>a97266f7-b062-11e7-b126-6b0f7a9b3379</RequestId>
+  </ErrorResponse>
+  """
+
+  setup do
+    bypass = Bypass.open
+    config = [
+      host: "http://localhost:#{bypass.port}",
+      region: "us-east-1",
+      access_key: "test_access",
+      secret: "test_secret"
+    ]
+
+    valid_email =
+      new()
+      |> from("guybrush.threepwood@pirates.grog")
+      |> to("elaine.marley@triisland.gov")
+      |> subject("Mighty Pirate Newsletter")

```

Comment:
Woah fancy, what are they? 😃

Reply:
The mightiest pirate who sailed the seas and his lass! A couple names from a game series a while ago. https://en.wikipedia.org/wiki/The_Secret_of_Monkey_Island

Definite recommend if you like those point and click kind of games. Think they remade a couple of them recently too. Like to dip my toes into my nostalgia when I can I guess.